build.sh

```diff
@@ -29,29 +31,30 @@ function ci() {
 }
 
 function ios_ci() {
-  ci $1 $1 iphonesimulator9.3 "platform=iOS Simulator,OS=9.3,name=iPhone 5" $2
+  ci $1 $2 iphonesimulator9.3 "platform=iOS Simulator,OS=9.3,name=iPhone 5s" $3
```

Comment:
The iPhone 5s is a 64-bit device that is compatible for the current set of reference images.